### ISSUE TITLE
Add support for BLAKE3 (b3sum files)

### ISSUE THREAD

**Issue by jarppiko** [cfv-project/cfv#41]
Hello, 

Big thanks for reviving the cfv project! I just installed it and it works great Have you considered adding [Blake 3](https://github.com/BLAKE3-team/BLAKE3) support? It is a very fast hash algo. 

My other feature request is an option to update the hash only if the file modify time is later than then checksum file. 

Cheers, 

J

**Comment by lxp** [comment]
Do I understand correctly that you want support for the file format used by the [b3sum](https://github.com/BLAKE3-team/BLAKE3#the-b3sum-utility) tool? Or do you have some other tool/file format in mind?

**Comment by jarppiko** [comment]
Yes, I think it is good `cfv` uses the same file format as the hash algos' native tools. `b3sum` is the tool for Blake 3. 

File format is:
```
# b3sum Dockerfile
7ba8fab84cfa0af5fcb47881a05e1dc1ca38da96dd044078881f886357d68883  Dockerfile
```

I have not tried Python [blake3](https://pypi.org/project/blake3/) yet.
